quickstart/fluent-package-v5-vs-td-agent.md

```diff
@@ -0,0 +1,58 @@
+# fluent-package v5 vs td-agent v4
+
+## Supported Platforms
+
+[Fluentd Project](https://www.fluentd.org) maintains stable packages for Fluentd and canonical plugins as Fluent Package \(the package is called `fluent-package`\)
+which is formerly known as `td-agent`.
+ `fluent-package` has v5 and LTS (Long Term Support) version.
```

Comment:
```suggestion
```

It seems that we should mention the channels in `Features`.

Reply:
I'll fix it.

Reply:
fixed.

Reply:
Was the fix reverted accidentally?

Reply:
Unexpected reverted, fixed again.